src/main/java/mate/academy/springboot/datajpa/dto/RequestCategory.java

```diff
@@ -0,0 +1,8 @@
+package mate.academy.springboot.datajpa.dto;
+
+import lombok.Data;
+
+@Data
+public class RequestCategory {
```

Comment:
```suggestion
public class CategoryRequestDto {
```

same for others Dtos